hermes/deletionPicker.go

```diff
@@ -0,0 +1,27 @@
+// Licensed under the Apache License, Version 2.0 (the "License")
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Author: Alicja Kwiecinska (kwiecinskaa@google.com) github: alicjakwie
```

Comment:
Remove these as we are not including Author lines as per Balazs's comment previously.
```suggestion
```